rules/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector.php

```diff
@@ -108,6 +108,10 @@ private function shouldSkip(ClassMethod $classMethod): bool
         }
 
         // skip magic methods - @see https://www.php.net/manual/en/language.oop5.magic.php
-        return $classMethod->isMagic();
+        if ($classMethod->isMagic()) {
+            return true;
+        }
+
+        return $classReflection->hasMethod('__call');
```

Comment:
Should this checks for invocations of __call() then?

Reply:
`__call` is common used, and running the check can be on specific directory, not the caller, I think currently, this is the safest way.